src/Admin/AbstractAdmin.php

```diff
@@ -1914,6 +1921,7 @@ final public function getTemplateRegistry(): MutableTemplateRegistryInterface
         if (false === $this->hasTemplateRegistry()) {
             throw new \LogicException(sprintf('Unable to find the template registry for admin `%s`.', static::class));
         }
+        \assert(null !== $this->templateRegistry);
```

Comment:
not sure about this, the hasTemplateRegistry already makes sure about it, maybe replace the if with this condition?

Reply:
Since we have a method, I think it's better to use it, and not modifying the code just to make psalm happy.